src/misc/person.v1.yaml

```diff
@@ -26,12 +26,37 @@ properties:
                       - Schekman, Randy
                       - VijayRaghavan, Krishnaswamy
                       - Li Wenhui
+            given_names:
+                title: Given names
+                description: |
+                    The given names of the person.
+
+                    For example:
+                      - Randy
+            surname:
+                title: Surname
+                description: |
+                    The surname of the person.
+
+                    For example:
+                      - Schekman
         required:
           - preferred
           - index
+          - last
     orcid:
         title: ORCID
         type: string
         pattern: ^[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]$
+    correspondence:
+        title: Correspondence Information
+        type: object
+        properties:
+            email:
+                title: Email address
+                description: This person's email for correspondence.
+            required:
+                - email
 required:
   - name
+  - correspondence
```

Comment:
This is not a candidate for being required as most of the BRE we have no need to store their email addresses.